swift/Workflow/Sources/WorkflowHost.swift

```diff
@@ -53,6 +53,19 @@ public final class WorkflowHost<WorkflowType: Workflow> {
 
     }
 
+    /// Update the input for the workflow. Will cause a render pass.
+    public func update(workflow: WorkflowType) {
+        rootNode.update(workflow: workflow)
+
+        // Treat the update as an "output" from the workflow as an external event to force a render pass.
```

Comment:
Nit: "as an" twice – intentional? Not sure how to parse.

Reply:
Whoops. Will update to hopefully make this more readable.